Assets/Scripts/Models/Furniture.cs

```diff
@@ -980,6 +972,33 @@ public string GetJobDescription()
         return string.Empty;
     }
 
+    public IEnumerable<string> GetAdditionalInfo()
+    {
+        yield return string.Format("Hitpoint 18 / 18");
+
+        if (PowerConnection != null)
+        {
+            bool hasPower = HasPower();
```

Comment:
No, IsOperating is a weird one, only in use when a furniture is doing a job or something. Here I'm showing the grid online/offline status. IsOperating would be ok so a smelter for instance.